hledger-lib/Hledger/Data/Account.hs

```diff
@@ -85,33 +108,47 @@ nullacct = Account
 -- The accounts are returned as a list in flattened tree order,
 -- and also reference each other as a tree.
 -- (The first account is the root of the tree.)
```

Comment:
In this (and maybe the next), let's explain a bit more that this calculates multiple inclusive and exclusive change amounts for each account, associated with the dates produced by the given date-bucketing function (usually representing the end dates of report subperiods)

Reply:
(It looks like this will enable non-contiguous report periods, which will be cool)

Reply:
Done